fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/repository/SocialMediaRepositoryTest.java

```diff
@@ -0,0 +1,128 @@
+package com.github.dedis.popstellar.repository;
```

Comment:
these tests are overalls not readable 😢 they're not bad at all, just not very readable.
I'll try to provide some comments to help make it cleaner 😄

Reply:
Yes, understandable 😅

I had a hard time writing the reactive tests, I almost never did it before.